src/controllers/account-controller.js

```diff
@@ -1,48 +1,213 @@
 const Accounts = require('../models/accounts');
-const {sendToTelegram} = require('../helpers/telegramBot');
-const {jsonErrors} = require('../helpers/exception-responses');
-const {ResponseREST} = require('../helpers/responses');
+const {StatusCodes} = require ('http-status-codes');
+const mongoose = require('mongoose');
+const {generateCode} = require('../helpers/generateCode');
+const {responseServer, responseNotFound, responseGeneral} = require('../helpers/response-result');
+const {sendEmail} = require('./email-controller');
+
+const validateExistsAccount = (request, response, next) => {
+    const {idAccount} = request.body;
+    Accounts.findById(idAccount, {_id:1})
+    .then(function (account) {  
+        if(account){
+            return next();
+        }
+        return responseGeneral(response, StatusCodes.BAD_REQUEST, "La cuenta no existe");
+    })
+    .catch(function (error){
+        return responseServer(response, error);
+    });
+}
+
+const validateExistsAccounts = (request, response, next) => {
+    const { idAccount, accountReported} = request.body;
+    Accounts.find({$or:[{_id:accountReported},{_id:idAccount}]}, {_id:1})
+    .then(function (accounts) {  
+        if(accounts.length == 2){
+            return next();
+        }
+        return responseGeneral(response, StatusCodes.BAD_REQUEST, "La cuenta o cuentas no existen");
+    })
+    .catch(function (error){
+        return responseServer(response, error);
+    });
+}
+
+const validateExistsUsernameEmail = (request, response, next) => {
+    const { email, username} = request.body;
+    Accounts.find({$or:[{username:username},{email:email}]}, {_id:1})
```

Comment:
Codacy has a fix for the issue: [Expected property shorthand.](https://app.codacy.com/gh/Martha-Miroslava/APIBiziidiDiidxaza/pullRequest?prid=8267260)
```suggestion
    Accounts.find({$or:[{username},{email:email}]}, {_id:1})
```